src/data/CoreData.tsx

```diff
@@ -46,6 +49,7 @@ export type CoreData = {
   locales: Record<BCP47LocaleCode, LocaleData>;
   territories: Record<TerritoryCode, TerritoryData>;
   writingSystems: Record<ScriptCode, WritingSystemData>;
+  variantTags: VariantTagData[]; // issue 6
```

Comment:
Use a hashmap here instead of an array. You see the other object types are using hashmaps to make retrieval fast.

```suggestion
  variantTags: Record<IANAVariantTag, VariantTagData>;
```